packages/cli/package.json

```diff
@@ -111,6 +111,8 @@
     "sucrase": "^3.20.2",
     "tar": "^6.1.2",
     "terser-webpack-plugin": "^5.1.3",
+    "ts-transform-import-path-rewrite": "^0.3.0",
+    "typescript": "~4.5.4",
```

Comment:
I'm thinking this should most likely be a peer dependency that includes all the TypeScript versions that we support in our usage, probably a `>=` range. Want to provide a lot of freedom in what TS versions are used in a project and encourage making it an explicit dependency in the repo root. In the end we're consuming TS declaration files that are not generated by our bundled version anyway, so feeling it makes sense at that level too.

Reply:
Makes sense!

Reply:
Do we have our minimum supported TS version documented anywhere? Proposed `>=4` as a starting point in ed2ac1fde3.